docs/src/main/asciidoc/openapi-swaggerui.adoc

```diff
@@ -385,6 +406,44 @@ Alternative paths are:
 Live reload of static OpenAPI document is supported during development. A modification to your OpenAPI document will be picked up on fly by Quarkus.
 ====
 
+== Changing the openapi version
+
+By default, when the document is generated, the openapi version used will be `3.0.3`. If you use a static file as mentioned above, the version in the file 
+will be used. You can also define the version in SmallRye using the following configuration:
+
+[source, properties]
+----
+mp.openapi.extensions.smallrye.openapi=3.0.2
+----
+
+This might be useful if your API go through a Gateway that needs a certain version.
+
+== Auto-generation of OperationId
+
+The https://swagger.io/docs/specification/paths-and-operations/[Operation Id] can be set using the `@Operation` annotation, and is in many cases useful when using a tool to generate a client stub from the schema. 
```

Comment:
Sorry for the nitpicking but you write `OperationId` in the title and then `Operation Id` here and twice below. Which one is it? Better be consistent.